openapi.yml

```diff
@@ -5569,6 +5596,143 @@ components:
+                                -
+                                    key: question_3
+                                    value: Postal Code
+                                -
+                                    key: question_4
+                                    value: V3B0K9
+                    hosts:
+                        -
+                            title: This is a host page
+                            sequence: 2
+                            id: 20
+                            data:
+                                -
+                                    fullname: Cam Wiebe
+                                    department: CTO
+                                    email: cwiebe@tg.com
+                                    profile_picture: 'https://tg.com/picture/jpg'
+                                -
+                                    fullname: Dan Souza
+                                    department: Dev
+                                    email: dsouza@tg.com
+                                    profile_picture: 'https://tg.com/picture/jpg'
+                    identities:
+                        -
+                            title: Tell us a bit about yourself
+                            id: 33
+                            sequence: 3
+                            data:
+                                email: gqueirz@tg.com
+                                company: TG
+                                fullname: Gabe Queiroz
+                                mobile: '+123123123'
+                    guest_signs:
+                        -
+                            title: Please read info about COVID-19
+                            id: 44
+                            sequence: 4
+                            data:
+                                status: signed
+                                document_name: COVID-19 - Gabriel Queiroz - 2020-01-01
+                                template_name: COVID-19
+                                document_url: 'https://tg.com/document/id'
+                        -
+                            title: Please read info about our security measures
+                            id: 55
+                            sequence: 5
+                            data:
+                                status: declined
+                                template_name: Security
+                    docusigns:
+                        -
+                            title: Please read our Docusign thingy
+                            id: 66
+                            sequence: 6
+                            data:
+                                status: signed
+                                envelope_id: 123123-123123123-123123123-123123123
+                                template_id: 123123123-123123123-123123123-123123
+                                template_name: Docusign thingy
+                                document_url: 'https://tg.com/document/id'
+                    branches:
```

Comment:
What is a branch here?
Is this a logical choice that someone makes?

I think `logic_branches`, `logical_branches`, or `user_choices` would be clearer